backend/src/Prelude/Ply.fs

```diff
@@ -233,3 +233,24 @@ module Map =
         })
       Map.empty
       dict
+
+// TODO : Add more functions
+module Result =
+  let mapSequentially
```

Comment:
I think I'd call this `map` - since there's only one, there's nothing to do seqentially.